lib/validators/schema.js

```diff
@@ -16,7 +16,18 @@ let zSchema = initializeZSchema();
  */
 function validateSchema (api) {
   // Choose the appropriate schema (Swagger or OpenAPI)
-  let schema = api.swagger ? openapi.v2 : openapi.v3;
+  let schema;
+
+  if(api.swagger){
+    schema = openapi.v2;
+  }else{
+    if(api.openapi.startsWith('3.1')){
+      schema = openapi.v31;
+      zSchema.setRemoteReference("https://json-schema.org/draft/2020-12/schema", schema, {ignoreUnresolvableReferences: true});
```

Comment:
Note that zSchema doesn't support this schema yet (https://github.com/zaggino/z-schema/issues/268)

Reply:
I guess we'll have to wait for that to be resolved?

Reply:
This added line is a workaround to wait for the zSchema issue to be resolved. But it's not an easy fix so I think you could merge it like that.

Reply:
Maybe it will be better to switch out from z-schema to ajv, which already supports draft 2020-12?